src/ngMessages/messages.js

```diff
@@ -257,7 +257,21 @@ var jqLite;
  * .some-message.ng-leave.ng-leave-active {}
  * ```
  *
- * {@link ngAnimate Click here} to learn how to use JavaScript animations or to learn more about ngAnimate.
+ * {@link ngAnimate See the ngAnimate docs} to learn how to use JavaScript animations or to learn
+ * more about ngAnimate.
+ *
+ * ## Displaying a default message
+ * If the ngMessages renders no inner ngMessage directive (that is to say when the key values does not
```

Comment:
The wording here does not sound right. Should it be "(that is to say when no value matches any of the ngMessage directives)"?

Reply:
Yeah, it's not correct. It should be something like "when a key in the ngMessages collection is truthy, but no matching message is defined"